### PR TITLE
Refactor API to the new structure

### PR DESCRIPTION
**Pull request by fpyshnyi**

### CODE REVIEW

File: src/main/java/com/icthh/xm/ms/timeline/service/mapper/XmTimelineMapper.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+    List<TimelineEvent> xmTimelineToTimelineEvent(List<XmTimeline> xmTimelines);
+
+    List<Timeline> xmTimelineToTimeline(List<XmTimeline> xmTimelines);
+
+    @Named("aggregateIdToEntityId")
+    default Long aggregateIdToEntityId(String aggregateId) {
+        if (aggregateId != null) {
+            try {
+                return Long.parseLong(aggregateId);
```
Comment (major13ua) [inline src/main/java/com/icthh/xm/ms/timeline/service/mapper/XmTimelineMapper.java:44]: Situation with NumberFormatException could be very harmful for service.
Proposal:
- before Long.parseLong use StringUtils.isNumber() to test and return null if not
- log.debug if aggregateId is not number
- log.error NumberFormatException

File: src/main/resources/config/liquibase/changelog/20230214000000_add_columns_for_new_structure.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <property name="jsonType" value="varchar" dbms="h2"/>
+    <property name="jsonType" value="jsonb" dbms="postgresql"/>
+    <property name="jsonType" value="text" dbms="oracle"/>
+
+    <changeSet id="20230214000000" author="fpyshnyi">
+        <addColumn tableName="xmtimeline">
+            <column name="source" type="varchar(255)"/>
```
Comment (major13ua) [inline src/main/resources/config/liquibase/changelog/20230214000000_add_columns_for_new_structure.xml:13]: chould it be 255 bytes or char?

Comment (fpyshnyi) [reply src/main/resources/config/liquibase/changelog/20230214000000_add_columns_for_new_structure.xml:13]: I'm sorry, but I don't understand this question. This field will have the same data type as most text fields in the xmtimeline table (PostgreSQL - varchar(255), Oracle - varchar2(255)).

Comment (major13ua) [reply src/main/resources/config/liquibase/changelog/20230214000000_add_columns_for_new_structure.xml:13]: for me OK if suitable for your needs.

**Review by major13ua — APPROVED** [review]

File: gradle.properties [hunk 3]
```diff
@@ -1,6 +1,6 @@
 rootProject.name=timeline
 profile=dev
-version=2.0.14
+version=2.0.15
```
Comment (pkev777) [inline gradle.properties:3]: Bump up major version

**Review by pkev777 — APPROVED** [review]

**Pull request merged**